visidata/loaders/postgres.py

```diff
@@ -45,7 +47,9 @@ def openurl_postgres(url, filetype=None):
                 dbname=dbname,
                 host=url.hostname,
                 port=url.port,
-                password=url.password)
+                password=url.password,
+                # Not sure if this is necessary
```

Comment:
If it's not necessary, let's remove it; otherwise the comment should be removed.

Reply:
Alright, removed in f60501428b6a5aafc73d213f5b6c351c67ceed12.